modules/__tests__/keyframes-test.js

```diff
@@ -45,7 +49,7 @@ describe('keyframes', () => {
     expect(document.createElement).not.to.have.been.called;
     expect(document.head.appendChild).not.to.have.been.called;
 
-    var name = keyframes({});
+    var name = keyframes({}, MyComponent);
```

Comment:
I have already done that [here](https://github.com/AnSavvides/radium/commit/e31e89a58124b4711328afadac1b017045a457b2#diff-af1a46446733ef5ab0fcce554afc942fR82).